src/unsafe-deep-dive/motivations/data-structures.md

```diff
@@ -0,0 +1,20 @@
+# Data Structures
+
+Some families of data structures, are impossible to create in safe Rust.
+
+- graphs
+- bit stuffing
```

Comment:
```suggestion
- bit twiddling
```

https://en.wikipedia.org/wiki/Bit_stuffing is a very specific technique in communication protocols, I think you're referring to "bit twiddling" https://en.wikipedia.org/wiki/Bit_manipulation

Reply:
Very good point. I wasn't entirely confident and it seems that lack of confidence has been proven valid. Fixed 65d60494863936af3a04b16f78052350e410f131